flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/runtime/stream/sql/SqlITCase.scala

```diff
@@ -870,6 +871,45 @@ class SqlITCase extends StreamingWithStateTestBase {
 
     assertEquals(List(expected.toString()), StreamITCase.testResults.sorted)
   }
+
+  @Test
+  def testProjectionWithManyColumns(): Unit = {
+
+    val env = StreamExecutionEnvironment.getExecutionEnvironment
+    val tEnv = StreamTableEnvironment.create(env)
+    StreamITCase.clear
+
+    // force code split
+    tEnv.getConfig.setMaxGeneratedCodeLength(1)
```

Comment:
remove this? 1000 columns will exceed the 64k limitation which will fail before the fix. This case is similar to the above `testVeryBigQuery()`.

Reply:
makes sense that functionally it will be the same after removing this explicit setup.

i think we might want to explicitly set it so that this unit test does not depend on the default value of `maxGeneratedCodeLength`, which hypothetically can change, similarly to the test `testSelectExpressionWithSplitFromTable()`.  Plus, we don't have any perf issue adding this.

what do you think?  i'm okay to remove it if you see that's not an issue.

Reply:
Thanks. I think it's fine to keep the config here.